src/sentry/eventstream/snuba.py

```diff
@@ -98,6 +99,44 @@ def insert(
         if unexpected_tags:
             logger.error("%r received unexpected tags: %r", self, unexpected_tags)
 
+        # HACK: We are putting all this extra information that is required by the
+        # post process forwarder into the headers so we can skip parsing entire json
+        # messages. The post process forwarder is currently bound to a single core.
+        # Once we are able to parallelize the JSON parsing and other transformation
+        # steps being done there we may want to remove this hack.
+        def encode_bool(value: Optional[bool]) -> str:
+            if value is None:
+                value = False
+            return str(int(value))
```

Comment:
Could we have encoding and decoding functions in the same place (like in the same module)? That would make it much easier to ensure they are still compatible with each other ?

Reply:
Moved it to the kafka backend only